frontend/SideDish/SideDish/ViewModel/MenusViewModel.swift

```diff
@@ -56,22 +69,8 @@ class MenusViewModel {
             return NSMutableAttributedString(string: "")
         }
     }
-    
-    func validate(pastPrice: String?) -> NSMutableAttributedString {
-        if let pastPrice = pastPrice {
-            let attributeString: NSMutableAttributedString =  NSMutableAttributedString(string: "\(pastPrice)원")
-            attributeString.addAttribute(NSAttributedString.Key.strikethroughStyle, value: 2, range: NSMakeRange(0, attributeString.length))
-            return attributeString
-        } else {
-            return NSMutableAttributedString(string: "")
-        }
-    }
-    
-    func setBadges(_ badges: [String]?) -> [String] {
-        if let badges = badges {
-            return badges
-        } else {
-            return []
-        }
-    }
+}
+
+extension MenusViewModel {
+    static let changeMenu = Notification.Name("changeMenu")
```

Comment:
MenusViewModel 이 아니라 Notification.Name으로 확장할 수 있습니다!
```suggestion
    static let changeMenu = Notification.Name("changeMenu")
    
 extension Notification.Name {
    static let changeMenu = Notification.Name("changeMenu")
} 

// called
NotificationCenter.default.addObserver(self, selector: #selector(sendMenuList), name: .changeMenu, object: menusViewModel)
```